kolibri/plugins/facility/assets/src/views/AllFacilitiesPage.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+
+  <KPageContainer>
+    <h1>{{ coreString('facilitiesLabel') }} </h1>
+    <CoreTable>
+      <thead slot="thead">
+        <tr>
+          <th>{{ coreString('nameLabel') }}</th>
+          <th>{{ coreString('classesLabel') }}</th>
+        </tr>
+      </thead>
+      <tbody slot="tbody">
+        <tr v-for="facility in facilities" :key="facility.id">
+          <td>
+            <KLabeledIcon icon="facility">
+              <KRouterLink
+                :text="facility.name"
+                :to="facilityLink(facility)"
+              />
+            </KLabeledIcon>
```

Comment:
`KRouterLink` has a new `icon` prop and slot. Also it uses `KLabeledIcon` internally now for that.

Once that change is merged to develop, this can become:

```
<KRouterLink
    :text="facility.name"
    :to="facilityLink(facility)
    icon="facility"
/>